### DISCUSSION TITLE
Table Export - non-null blank values are not quoted

### DISCUSSION THREAD

**Discussion by mlowe108** [dbeaver/dbeaver#9432] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: MacOS Catalina 10.15.6
- DBeaver version: 7.1.4
- Additional extensions

#### Connection specification:
- Database name and version: IBM Db2 for z/OS 
- Driver name: db2jcc_license_cisuz.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:

My table has columns all defined as NOT NULL. Some of the columns contain all spaces as a valid value. For example, a column defined as CHAR(8) NOT NULL contains eight spaces. The problem is that EXPORT does NOT delimit the spaces with quotes. As a result, when I try to import the .csv into Db2 for Linux, the Db2 LOAD utility fails as it sees null values in the input file.

As an aside, DBeaver successfully imports the file, so must compensate somehow.

#### Include any warning/errors/backtraces from the logs

Here is a sample record with the problem:

0,                ,20160915,20160915,11372400,SCM     ,SCMFLT, ,TEST0SIC,1,XX:00000000                                 ,15/09/2016 11:37

Fields 2 and 8 contain spaces but appear to be null.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please use **quote always** setting of CSV export.
![dbeaver_4vJMtn12rm](https://user-images.githubusercontent.com/51405061/89172198-e7d63600-d58a-11ea-838c-02ad94750269.png)

Please reopen the ticket if it doesn't help.

**Comment by mlowe108** [comment]
Thanks, problem fixed!
